tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -790,7 +792,6 @@ public void onZoomEnd( float scale, Origination origin ) {
 
   @Override
   public void onDetailLevelChanged( DetailLevel detailLevel ) {
-    requestRender();
```

Comment:
Here, the requestRender is useless, since it is cancelled right away in `mTileCanvasViewGroup.updateTileSet( detailLevel );` and then called.